applications/concept-cat/src/main/java/no/ccat/dto/ConceptPublisherMessage.java

```diff
@@ -0,0 +1,23 @@
+package no.ccat.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@EqualsAndHashCode(callSuper = false)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+public class ConceptPublisherMessage extends HarvestTriggerMessage {
```

Comment:
Consider a more descriptive and to-the-point name

Reply:
removed and replaced all with JsonNode